### PR TITLE
Move tests back into `frameworks/` folder

### PR DESCRIPTION
**Pull request by RasmusWL**
For https://github.com/github/codeql/pull/8883

### CODE REVIEW

File: python/ql/test/experimental/meta/MaDTest.qll [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import python
+private import semmle.python.dataflow.new.DataFlow
+private import semmle.python.dataflow.new.internal.PrintNode
+private import semmle.python.frameworks.data.ModelsAsData
+// need to import Frameworks to get the actual modeling imported
+private import semmle.python.Frameworks
+// this improt needs to be public to get the query predicates propagated to the actual test files
```
Comment (erik-krogh) [inline python/ql/test/experimental/meta/MaDTest.qll]:
```suggestion
// this import needs to be public to get the query predicates propagated to the actual test files
```

**Comment by erik-krogh** [comment]
I like it.  

But does the `MaDTest.qll` file belong in an `experimental/` folder? How about putting it in `TestUtilities`?

**Comment by RasmusWL** [comment]
> But does the `MaDTest.qll` file belong in an `experimental/` folder? How about putting it in `TestUtilities`?

I hoped no one would see that :see_no_evil: The answer is no, it should not be there. But the other test library code should not be there either :grimacing:

If you care about this, I can move it, but I am inclined to just leave it for some proper cleaning :smile:

**Comment by erik-krogh** [comment]
> If you care about this, I can move it, but I am inclined to just leave it for some proper cleaning :smile: 

I'll merge this. Then I'll leave that refactor for you at a later time.

**Pull request merged**